dive_core/capture_data.cpp

```diff
@@ -1227,6 +1240,13 @@ const DiveVector<SubmitInfo> &CaptureData::GetSubmits() const
     return m_submits;
 }
 
+//--------------------------------------------------------------------------------------------------
+const std::vector<std::unique_ptr<DiveAnnotationProcessor::SubmitInfo>> &
+CaptureData::GetGfxrSubmits() const
+{
+    return std::move(m_gfxr_submits);
```

Comment:
why do you use std::move here? isn't `GetGfxrSubmits() const` a read-only access?

Reply:
Removed.